lib/datetime_helper.rb

```diff
@@ -1,6 +1,6 @@
-require "westfield_datetime_helper/version"
+require "datetime_helper/version"
 
-module WestfieldLabs
+module Labs
```

Comment:
I think either `Labs` need to be removed or the gem name should be changed to `labs-datetime_helper` See http://guides.rubygems.org/name-your-gem/